account_menu_invoice_refund/hooks.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, _
+from odoo.exceptions import UserError
+from odoo.addons.account.models.account_payment import account_abstract_payment
+
+MAP_INVOICE_TYPE_PARTNER_TYPE = {
+    'out_invoice': 'customer',
+    'out_refund': 'customer',
+    'in_invoice': 'supplier',
+    'in_refund': 'supplier',
+}
+
+
+def post_load_hook():
```

Comment:
It works so this seems correct, but why not putting it on the account_abstract_payment class?

Reply:
You mean, why hot inherit account.abstract.payment?
I want to remove the checking, I need use hook. I don't know how to do it any other way though.

Reply:
:+1: